src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -93,14 +91,33 @@ public Test()
 
             public Func<ImmutableArray<Diagnostic>, Diagnostic?>? DiagnosticSelector { get; set; }
 
-            public override async Task RunAsync(CancellationToken cancellationToken = default)
+            protected override async Task RunImplAsync(CancellationToken cancellationToken = default)
             {
                 if (DiagnosticSelector is object)
                 {
                     Assert.True(CodeFixTestBehaviors.HasFlag(Testing.CodeFixTestBehaviors.FixOne), $"'{nameof(DiagnosticSelector)}' can only be used with '{nameof(Testing.CodeFixTestBehaviors)}.{nameof(Testing.CodeFixTestBehaviors.FixOne)}'");
                 }
 
-                await base.RunAsync(cancellationToken);
+                var (analyzerConfigSource, _) = CodeFixVerifierHelper.ConvertOptionsToAnalyzerConfig(DefaultFileExt, EditorConfig, Options);
+                if (analyzerConfigSource is object)
+                {
+                    if (_analyzerConfigIndex is null)
+                    {
+                        _analyzerConfigIndex = TestState.AnalyzerConfigFiles.Count;
+                        TestState.AnalyzerConfigFiles.Add(("/.editorconfig", analyzerConfigSource));
+                    }
+                    else
+                    {
+                        TestState.AnalyzerConfigFiles[_analyzerConfigIndex.Value] = ("/.editorconfig", analyzerConfigSource);
+                    }
+                }
+                else if (_analyzerConfigIndex is { } index)
+                {
+                    _analyzerConfigIndex = null;
+                    TestState.AnalyzerConfigFiles.RemoveAt(index);
```

Comment:
i don't quite get what's going o nhere.

Reply:
The code fix tests got more strict, and now verify the before/after contents of **.editorconfig**. It's expecting the inputs to be declared in `TestState.AnalyzerConfigFiles`, but we can't do that since the content is generated dynamically.